.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The .idea directory should not be part of the project.

Reply:
Removed.

Reply:
It is not.